src/index.mjs

```diff
@@ -0,0 +1,11 @@
+import { createRequire } from 'module';
```

Comment:
Not that is makes much difference, but could you change this to:

```js
import mod from './index.js';

export const { ManagementClient, AuthenticationClient } = mod;
export default { ManagementClient, AuthenticationClient }
```

(and check it still fixes your issue)

This works for Node 19 and looks a bit simpler - also i don't think we need the doc (I should probably remove this from the other index too)